components/SplitDisplay.vue

```diff
@@ -54,6 +60,11 @@ export default class SplitDisplay extends Vue {
   @Prop()
   currentAttemptNumber?: number;
 
+  @Prop()
+  splitIndex!: number;
+
+  segments: Array<Segment> = store.state.splitFile.Run.Segments.Segment;
```

Comment:
Minor coding style suggestion (Usually I use "Array" only when the type inside is complex, like `Array<number | null>`)
```suggestion
  segments: Segment[] = store.state.splitFile.Run.Segments.Segment;
```